bundles/org.openhab.binding.orbitbhyve/src/main/java/org/openhab/binding/orbitbhyve/internal/handler/OrbitBhyveBridgeHandler.java

```diff
@@ -210,6 +211,9 @@ public List<OrbitBhyveDevice> getDevices() {
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         "Get devices returned response status: " + response.getStatus());
             }
+        } catch (JsonSyntaxException e) {
+            logger.info("Exception parsing devices json: {}", e.getMessage(), e);
```

Comment:
DEBUG level would be more appropriate, especially if you log the full stack trace.

Reply:
Yes, I think that's fair. Previously before the exception was caught, it would just fail silently. Now the bridge will go offline, which will be a clue to enable debug.

Note in my latest commit I'm also catching `JsonSyntaxException` in the getDevice method because it will fail in the same way.